inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/page/workload/WorkloadPageMenuItem.java

```diff
@@ -94,10 +91,11 @@ public boolean applies()
 
         // Visible if the current user is a curator or project admin
         User user = userRepo.getCurrentUser();
+
         return (projectService.isCurator(project, user)
             || projectService.isProjectAdmin(project, user))
             && WebAnnoConst.PROJECT_TYPE_ANNOTATION.equals(project.getMode())
-            && workloadProperties.isWorkloadManagerActive();
+            && !workloadAndWorkflowService.getWorkloadManager(project).equals("Default monitoring page");
```

Comment:
I do not like literal strings, please use constants

Reply:
Internationalizable strings should never be used for logic anyway. See e.g. how the `ActionBarExtensionPointImpl`and  `FeatureSupportRegistryImpl` implement an extension points and how they use the `ExtensionPoint_ImplBase` base class. Then also how e.g. `ActionBarExtension` plugs into that extension point and how it uses the `Extension` base class which defines a `getId()` method. Please use these base classes (`ExtensionPoint_ImplBase` and `Extension` unless there is a good reason not to).